src/Behaviour.js

```diff
@@ -0,0 +1,11 @@
+class Behaviour {
+  constructor(trigger, keyword, action) {
+    this.trigger = trigger;
+    this.keyword = keyword;
+    this.action = action;
+  }
+}
+
+module.exports = {
+  Behaviour
```

Comment:
How is this different from the original `src/behavior.js` file?

Reply:
It is not, I was not sure how to include behaviors into the closed loop of this framework, let's have a discussion on that perhaps?